sonic_sfp/sff8436.py

```diff
@@ -16,7 +16,7 @@
     import types
     from math import log10
     from sff8024 import type_of_transceiver
-    from sffbase import sffbase
+    from .sffbase import sffbase
```

Comment:
Why are these dots necessary now? What was the error you were seeing?

Reply:
I was getting import error like **module not found**.

Reply:
Dot module supports both Python 2 and Python 3 using explicit relative import methods. Added comments wherever needed